src/Microsoft.TestPlatform.PlatformAbstractions/common/Process/ProcessHelper.cs

```diff
@@ -1,26 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Helpers
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Helpers
 {
     using System;
     using System.Collections.Generic;
     using System.Diagnostics;
     using System.IO;
     using System.Reflection;
-    using System.Threading;
-    using System.Threading.Tasks;
 
-    using Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Helpers.Interfaces;
-    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Helpers.Interfaces;
 
     /// <summary>
     /// Helper class to deal with process related functionality.
     /// </summary>
-    internal class ProcessHelper : IProcessHelper
+    public class ProcessHelper : IProcessHelper
```

Comment:
Does it help to make this `IProcessHelper<T>`? If the consumer is aware of `Process`, it may as well call the `ProcessHelper<Process>` (may be do not require to do type casts).